docs/src/admin-manual/iac.md

```diff
@@ -0,0 +1,152 @@
+corsOrigin = "http://localhost"
+
+[api.auth]
+# Enable API authentication. OIDC is the only protocol supported at the
+# moment. When enabled the API verifies the access token submitted with
+# each request. The API client is responsible for obtaining an access
+# token from the provider.
+enabled = true
+
+[api.auth.oidc]
+# OIDC provider URL. Required when auth. is enabled.
+providerURL = "http://keycloak:7470/realms/artefactual"
+# OIDC client ID. The client ID must be included in the `aud` claim of
+# the access token. Required when auth. is enabled.
+clientID = "enduro"
+
+[api.auth.oidc.abac]
+# Enable Attribute Based Access Control (ABAC). If enabled, the API will
+# check a configurable multivalue claim against required attributes based
+# on each endpoint configuration.
+enabled = true
+# Claim path of the Enduro attributes within the access token. If the claim
+# path is nested then include all fields separated by `claimPathSeparator`
+# (see below). E.g. "attributes.enduro" with `claimPathSeparator = "."`.
+# Required when ABAC is enabled.
+claimPath = "enduro"
+# Separator used to split the claim path fields. The default value of "" will
+# try to match the claim path as-is to a top-level field from the access token.
+claimPathSeparator = ""
+# Add a prefix to filter the values of the configured claim. If the claim
+# contains values unrelated to Enduro's ABAC, the values relevant to Enduro
+# should be prefixed so they are the only values used for access control.
+# For example, a claim with values ["enduro:*", "unrelated"] will be filtered
+# to a value of ["*"] when `claimValuePrefix = "enduro:"`. The default "" will
+# not filter any value.
+claimValuePrefix = ""
+
+[api.auth.ticket.redis]
+# Redis URI to store a ticket used to set a websocket connection.
+address = "redis://redis.enduro-sdps:6379"
+# Prefix used as part of the ticket keys in Redis.
+prefix = "enduro"
+```
+
+## Dashboard configuration
+
+The following environment variables can be used to configure the dashboard:
+
+```txt
+VITE_OIDC_ENABLED
+VITE_OIDC_AUTHORITY
+VITE_OIDC_CLIENT_ID
+VITE_OIDC_REDIRECT_URI
+VITE_OIDC_EXTRA_SCOPES
+VITE_OIDC_ABAC_ENABLED
+VITE_OIDC_ABAC_CLAIM_PATH
+VITE_OIDC_ABAC_CLAIM_PATH_SEPARATOR
+VITE_OIDC_ABAC_CLAIM_VALUE_PREFIX
+```
+
+They must match the ones configured in the API. `VITE_OIDC_AUTHORITY` has to be
+the same OIDC provider URL and `VITE_OIDC_CLIENT_ID` needs to be the same or a
+trusted client. This client (and the one used in the API configuration, if they
+are not the same) must be included in the `aud` claim from the access token,
+
+The authorization flow will request the `openid email profile` scopes by
+default. If needed, `VITE_OIDC_EXTRA_SCOPES` can be used to request additional
+scopes. The ABAC variables will work in the same way as they do in the API,
+they are explained in detail in the API configuration comments above.
+
+These environment variables must be set at build time or they can be replaced
```

Comment:
you say "must", and then follow with an "or" - is the or a bad outcome if you don't do the _must_? Or is it an alternative? If it's an alternative, then I would change the "must" to "can" - _must_ is typically used as an imperative (you must do this, or else...) 

> These environment variables can be set at build time, or they can be replaced in the final assets.